src/pe/debug.rs

```diff
@@ -1,15 +1,17 @@
 use crate::error;
+use alloc::vec::Vec;
 use scroll::{Pread, Pwrite, SizeWith};
 
 use crate::pe::data_directories;
 use crate::pe::options;
 use crate::pe::section_table;
 use crate::pe::utils;
 
-#[derive(Debug, PartialEq, Copy, Clone, Default)]
+#[derive(Debug, PartialEq, Clone, Default)]
 pub struct DebugData<'a> {
-    pub image_debug_directory: ImageDebugDirectory,
+    pub image_debug_directories: Vec<ImageDebugDirectory>,
```

Comment:
do you know if there is a maximum image debug directories possible? if yes, perhaps we should make this an array and put it back to `Copy`?

Reply:
Yes, the maximum entry is defined as 21.
https://github.com/winsiderss/systeminformer/blob/6f97511ad7bb0c8807d99a3151494f55d0e21fc8/phlib/include/mapimg.h#L714

Reply:
Index itself is maximum 21 but it isn't sorted that way, I mean theres some gaps between entries, and some of "gaps" aren't even defined as "reserved" entry. So I think the best way is to implement iterator for this, as well as how we do in unwind table and rich metadatas.